example/lib/main.dart

```diff
@@ -77,7 +77,40 @@ class _MyAppState extends State<MyApp> {
                           MainPageStrings.defaultCheckedPermission,
                         );
                 },
-              )
+              ),
+              ButtonWidget(
+                onPressed: () {
+                  UserLocationPlugin.initializePlugin;
+                },
+                buttonText: MainPageStrings.initializePluginButtonText,
+              ),
+              ButtonWidget(
+                onPressed: () {
+                  UserLocationPlugin.startListeningLocation;
+                },
+                buttonText: MainPageStrings.bottonLocationText,
+              ),
+              StreamBuilder(
+                stream: UserLocationPlugin.locationEventChannelStream,
+                builder: (
+                    BuildContext context,
+                    AsyncSnapshot<dynamic> snapshot,
```

Comment:
Try to define the type of data that you're going to use in StreamBuilder, an try to not use dynamic specify the correct type here

Reply:
I can't do that. I'm getting the stream from a method of the EventChannel, and it's defined to return a Stream<dynamic>